foundations/html_css/flexbox/flexbox-alignment.md

```diff
@@ -48,24 +48,17 @@ Take your time going through the reading. There will be some review of the items
 2. [Aligning Items in a Flex Container](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container) goes into more depth on the topic of axes and `align-items` vs `justify-content`.
 3. [Typical use cases of Flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Typical_Use_Cases_of_Flexbox) is one more MDN article that covers some more practical tips. Don't skip the interactive sections! Playing around with this stuff is how you learn it!
 4. The [CSS Tricks "Guide to Flexbox"](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) is a classic. There isn't any new information for you here, but the images and examples are super helpful. This one is a great cheat sheet that you'll probably return to often. (Keep it handy for the practice exercises!)
+5. Go back to our [CSS exercises repository](https://github.com/TheOdinProject/css-exercises) (don't forget that the instructions are in the README). Do the exercises in the 'flex' directory in the following order:
+    * 01-flex-center
+    * 02-flex-header
+    * 03-flex-header-2
+    * 04-flex-information
+    * 05-flex-modal
+    * 06-flex-layout
+    * 07-flex-layout-2
+    * It may take you a while to get through all of them, and the difficulty ramps up as you progress. Stick with it! If you can get through all of them, then you will be in _really_ good shape moving forward.
```

Comment:
Nitpick: do you think it might be better to take that last bullet out ("It may take...") and instead inline it with the numbered list item? E.g.:

```
5. ... Do the exercises listed below in the order that they are listed. It may take you a while....
```
Not overly fussed over it, but I'm thinking maybe if users might start the exercises without reading them all listed there and thus not seeing the last bullet item. Though this could be an issue that doesn't exist/isn't significant, so I leave it up to you; I'm good either way, so I'm approving it now regardless.

Reply:
I was debating taking off that last instruction item for a while but hadn't thought to just tack it into the number list item. I like it! I'll make the change.